### PR TITLE
adds xpi config to balrog & beetmover workers

### PR DESCRIPTION
**Pull request by gabrielbusta**
Adds xpi config to balrog & beetmover workers.

### CODE REVIEW

File: beetmoverscript/docker.d/init_worker.sh [hunk 1]
```diff
@@ -40,6 +40,21 @@ case $COT_PRODUCT in
...
     ;;
+  xpi)
+    case $ENV in
+      dev|fake-prod)
+        test_var_set 'DEP_ID'
+        test_var_set 'DEP_KEY'
+        ;;
+      prod)
+        test_var_set 'RELEASE_ID'
+        test_var_set 'RELEASE_KEY'
```
Comment (escapewindow) [inline beetmoverscript/docker.d/init_worker.sh:51]: These match up with sops 👍

File: beetmoverscript/docker.d/worker.yml [hunk 2]
```diff
@@ -99,6 +99,22 @@ bucket_config:
...
             geckoview: 'maven-default-s3-upload-bucket-1705kkmo2aj3c'
         dummy: {}
+      'COT_PRODUCT == "xpi" && (ENV == "dev" || ENV == "fake-prod")':
+        dep:
+          credentials:
+            id: { "$eval": "DEP_ID" }
+            key: { "$eval": "DEP_KEY" }
+          buckets:
+            xpi: 'net-mozaws-stage-delivery-archive'
+          url_prefix: 'https://ftp.stage.mozaws.net'
```
Comment (escapewindow) [inline beetmoverscript/docker.d/worker.yml:109]: We may want to add a `dummy: {}` line -- I'm not 100% sure but I think this lets us test basic sanity checks without actually pushing to a bucket. I *think* we have that for every pool, but we'd at least want the dev/fake-prod pools.

**Review by escapewindow — APPROVED** [review]
r+ with changes, thank you!

File: beetmoverscript/docker.d/worker.yml (hunk 2 above)
Comment (gabrielbusta) [reply beetmoverscript/docker.d/worker.yml:109]: OK. I added it.

**Pull request merged**